content/influxdb/cloud/account-management/data-usage.md

```diff
@@ -20,9 +20,9 @@ right corner of your {{< cloud-name "short" >}} user interface (UI) and select *
 Find data usage information for the time frame specified in the drop-down at the top of the Usage page.
 
 - **Data In:** Total data in MB written to your {{< cloud-name "short" >}} instance.
-- **Data Out:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
+- **Storage:** Total disk usage in gigabytes.
 - **Query Count:** Total number of individual query operations, which include queries, tasks (alerts, notifications) and Data Explorer activity.
-- **Storage Usage:** Total disk usage in gigabytes.
+- **Data Out:** Total data in MB sent as responses to queries from your {{< cloud-name "short" >}} instance.
 - **API Request Count:** The total number of query and write API requests received during the specified time frame.
```

Comment:
We do not have API Request Count anymore on our usage page.
![image](https://user-images.githubusercontent.com/10112209/119890007-24916600-beec-11eb-95ce-93494a650143.png)